src/app/redux/tests/global.test.js

```diff
@@ -19,7 +17,7 @@ describe('global reducer', () => {
         const state = Immutable.fromJS(require('./global.json'));
         //const action = {type: 'global/RECEIVE_STATE', payload: state};
         expect(
-            reducer(undefined, actions.receiveState(state))
+            reducer(undefined, {type: 'global/RECEIVE_STATE', payload: state})
```

Comment:
why is `undefined` being passed as an argument

Reply:
i didn't write this test & that's a good question :)

Reply:
@valzav ?

Reply:
discuss in https://github.com/steemit/condenser/issues/2038

Reply:
because when reducer is called with undefined state it will use the default one